src/stores/Ethplorer.ts

```diff
@@ -0,0 +1,29 @@
+import { observable } from 'mobx';
+import { TokenInfo, Holder } from '../types';
+
+import RootStore from 'stores/Root';
+import { getTokenInfo, getTopTokenHolders } from '../provider/ethplorer';
+
+interface DefaultTokenInfo {
+    holdersCount?: number;
+}
+
+export default class EthplorerStore {
+    @observable tokenInfo: TokenInfo | DefaultTokenInfo = {};
+    @observable holders: Holder[] = [];
+    rootStore: RootStore;
+
+    constructor(rootStore) {
+        this.rootStore = rootStore;
+        this.getTokenInfo();
+        this.getHolders(20);
+    }
+
+    private async getTokenInfo(): Promise<void> {
+        this.tokenInfo = await getTokenInfo();
```

Comment:
maybe you can just get the address of the token directly form the config store here.